apps/graph/base.de.i18n

```diff
@@ -31,3 +31,11 @@ DerivativeFunctionColumn = "Spalte der Ableitungsfunktion"
 HideDerivativeColumn = "Ableitungsfunktion ausblenden"
 AllowedCharactersAZaz09 = "Erlaubt: A..Z, a..z, 0..9, _"
 ReservedName = "Reserviertes Wort"
+FunctionColorBlack = "schwarz"
```

Comment:
Is it possible to start the colors with a capital letter?

Reply:
I don't know how to do that :rofl: no i'm joking but you think this pr is still valid ? bc it is not very compact  :thinking: as said @RedGl0w

Reply:
> I don't know how to do that

Just change the text...

Reply:
For the name of the colors, instead of somehow duplicating them, couldn't you rather use the pythonColors ?

Reply:
I think it is better to move the Python colour names in the shared i18n file in this case to can build without the Python app.